AxonDeepSeg/download_model.py

```diff
@@ -14,9 +14,11 @@ def download_model(model='generalist', model_type='light', destination=None):
     
     model_suffix = 'light' if model_type == 'light' else 'ensemble'
     full_model_name = f'{MODELS[model]["name"]}_{model_suffix}'
+
     if destination is None:
-        model_destination = Path(f"AxonDeepSeg/models/{full_model_name}")
+        model_destination = Path(".") / str(f"{full_model_name}")
```

Comment:
```suggestion
        model_destination = Path(".") / full_model_name")
```

Reply:
btw this if statement is never true when using the CLI because it defaults to `.`

Reply:
>btw this if statement is never true when using the CLI because it defaults to .

Yes I'm aware - but it could be called directly via python, so I left it